sql.go

```diff
@@ -38,6 +38,10 @@ func (u UUID) Value() (driver.Value, error) {
 // a longer byte slice or a string will be handled by UnmarshalText.
 func (u *UUID) Scan(src interface{}) error {
 	switch src := src.(type) {
+	case UUID: // support gorm convert from UUID to NullUUID
+		u = &src
```

Comment:
This is almost certainly not right. It changes the pointer `u`'s value, but not the UUID it points to. `*u = src`.

Reply:
done